python/python/glide/async_commands/cluster_commands.py

```diff
@@ -1076,67 +1076,78 @@ async def scan(
         match: Optional[TEncodable] = None,
         count: Optional[int] = None,
         type: Optional[ObjectType] = None,
+        allow_non_covered_slots=False,
```

Comment:
```suggestion
        allow_non_covered_slots: bool=False,
```

Reply:
It complains on it. It's direct, python gets mad it's redundant.
(I started with you suggest)

Reply:
No its not, another context. fixed.